torchao/dtypes/nf4tensor.py

```diff
@@ -40,6 +40,10 @@ def noop_detach(func, *args, **kwargs):
 def _to_copy(func, *args, **kwargs):
     return args[0][0].get_original_weight().to(args[1]['dtype'])
 
+@implements([torch.ops.aten.to.dtype])
```

Comment:
does this have to be in this file as opposed to the tutorial?

Reply:
No, I wanted to add coverage for this (and thought I already did). We need a CI.